targets/Program.cs

```diff
@@ -53,16 +53,7 @@ public static async Task Main(string[] args)
                 var source = Path.GetFullPath("./MinVer/bin/Release/");
                 var version = Path.GetFileNameWithoutExtension(Directory.EnumerateFiles(source, "*.nupkg").First()).Split("MinVer.", 2)[1];
 
-                File.WriteAllText(
-                    Path.Combine(testProject, "global.json"),
-@"{
-  ""sdk"": {
-    ""version"": ""3.1.300""
-  }
-}
-"
-                    );
-
+                await RunAsync("dotnet", "new globaljson --sdk-version 3.1.300", testProject);

```

Comment:
Without the --force option, it will fail if the file already exists. The current code just overwrites the file

Reply:
@thomaslevesque thanks for clarifying. Given that the directory is already ensured to be empty on line 51, we can do without the `--force` option.